torchmultimodal/transforms/bert_text_transform.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List
+
+from torchtext.transforms import (
+    AddToken,
+    BERTTokenizer,
+    Sequential,
+    StrToIntTransform,
+    ToTensor,
+)
+
+
+class BertTextTransform:
+    def __init__(
+        self,
+        vocab_file: str = "https://huggingface.co/distilbert-base-uncased/resolve/main/vocab.txt",
+        do_lower_case: bool = True,
```

Comment:
Where do those defaults come from? And what do they mean? Explain in the docstring?